mgmt/src/frr/frrmi/mod.rs

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+
+// An FRR config reloader object
```

Comment:
What does `frrmi` stand for (I guess “FRR Management Interface”)? Could we maybe expand it here?